File: lib/dapp/dimg/image/dimg.rb
module Dapp
  module Dimg
    module Image
      class Dimg < Stage
        def export!(export_name = name)
          super(export_name)
        end

        def tag!(tag_name = name)
          super(tag_name)
        end
      end
    end # Image
  end # Dimg
end # Dapp

File: pkg/image/dimg.go
package image

type Dimg struct {
	*Stage
}

func NewDimgImage(fromImage *Stage, name string) *Dimg {
	return &Dimg{Stage: NewStageImage(fromImage, name)}
}

func (i *Dimg) Tag() error {
	return i.Stage.Tag(i.name)
}

func (i *Dimg) Export() error {
	return i.Stage.Export(i.name)
}
